OmiseSwift/Models/FirstDigits.swift

```diff
@@ -0,0 +1,45 @@
+import Foundation
+
+public struct FirstDigits: Equatable {
+    public let firstDigits: String
+    
+    public init?(firstDigitsString: String) {
+        guard firstDigitsString.count == 6 &&
```

Comment:
I'm thinking about removing the checking for the length. Since there's a chance that this requirement would be changed in the future. Also for the Last Digits too